### PR TITLE
SLVS-1344 Unify styling for project and organization list

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**

### CODE REVIEW

File: src/ConnectedMode/UI/Resources/Styles.xaml [hunk 1]
```diff
@@ -87,4 +87,18 @@
         <Setter Property="BaselineAlignment" Value="TextBottom" />
     </Style>
 
+    <DataTemplate x:Key="NameAndKeyListBoxItem">
```
Comment (vnaskos-sonar) [inline src/ConnectedMode/UI/Resources/Styles.xaml:90]: Not sure if the `DataTemplate` belongs to the `Styles.xaml` it feels irrelevant, can it be extracted to another file?

File: src/ConnectedMode/UI/Resources/Styles.xaml [hunk 2]
```diff
@@ -87,4 +87,18 @@
...
+        <StackPanel Orientation="Horizontal">
+            <TextBlock Text="{Binding Name}" FontWeight="Bold" Margin="0,0,5,0" />
+            <TextBlock>
```
Comment (vnaskos-sonar) [inline src/ConnectedMode/UI/Resources/Styles.xaml:93]: As you also mentioned, we can make this text gray.

```suggestion
            <TextBlock Foreground="{DynamicResource {x:Static vsShell:VsBrushes.GrayTextKey}}">
```

File: src/ConnectedMode/UI/Resources/Styles.xaml (hunk 1 above)
Comment (georgii-borovinskikh-sonarsource) [reply src/ConnectedMode/UI/Resources/Styles.xaml:90]: I think it's fine because this template is used for display (ie styling)

Comment (vnaskos-sonar) [reply src/ConnectedMode/UI/Resources/Styles.xaml:90]: Not exactly, because it contains details about the layout and the data. I am not sure if this will have an impact on the other xaml files that import `Styles.xaml`. @gabriela-trutan-sonarsource ?

Comment (gabriela-trutan-sonarsource) [reply src/ConnectedMode/UI/Resources/Styles.xaml:90]: I think it is safe to add DataTemplate to Styles.xaml. It should not impact any other file, because it has a key defined. So as long as no control references that key, it will have no impact.

**Review by vnaskos-sonar — APPROVED** [review]
